src/core/lib/gpr++/README.md

```diff
@@ -0,0 +1,16 @@
+# GPR++ - Google Portable Runtime for C++
+
+The files in this directory contain various utility code for C++ code.
+None of this code is gRPC-specific; anything here may also be useful
+for other open source projects written in C++.
+
+Note that this is one of the few places in src/core where we allow
+the use of portability macros.
+
+Note that this is the only place in src/core where we allow
```

Comment:
This should be libc++ right? and we would need to statically link libc++ first before this is true. (libstdc++ can't be statically linked with gRPC because of licensing issues and we can't dynamically link because of platform issues)

Reply:
As per our conversation, we can't use either libc++ or libstdc++ right now.  But we can use things from `std::` as long as we're careful to only use template code that does not require linking against either of those libraries.

Reply:
Ignore the above comment.